website/docs/docs/build/unit-tests.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Unit tests"
+sidebar_label: "Unit tests"
+description: "Read this tutorial to learn how to use unit tests on your SQL models."
+search_weight: "heavy"
+id: "unit-tests"
+keywords:
+  - unit test, unit tests, unit testing, dag
+---
+:::note closed beta
+
+Unit testing is currently in closed beta for dbt Cloud accounts that have updated to a [versionless environment](/docs/dbt-versions/upgrade-core-in-cloud).
+
+It is available now as an alpha feature for dbt Core v1.8 users.
+
+:::
+
+Historically, the test coverage capabilities of dbt were limited to “data” tests &dash; testing the quality of input data or the shape of the resulting datasets &mdash; that could only be executed *after* a model had been built. 
+
+Now, we are introducing a new type of test to dbt - unit tests. In software programming, unit tests validate small portions of your functional code, and they work much the same way here. Unit tests allow you to validate your SQL modeling logic on a small set of static inputs _before_ you materialize your full model in production. Unit tests enable test-driven development, benefiting developer efficiency and code reliability. 
+
+Let’s say you’re creating a new `dim_customers` model with a field `is_valid_email_address`, that calculates whether or not the customer’s email is valid: 
+
+```sql
+with customers as (
+
+    select * from {{ ref('stg_customers') }}
+
+),
+
+accepted_email_domains as (
+
+    select * from {{ ref('top_level_email_domains') }}
+
+),
+	
+check_valid_emails as (
+
+    select
+        customers.customer_id,
+        customers.first_name,
+        customers.last_name,
+				coalesce (regexp_like(
+            customers.email, '^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}$'
+        )
+        = true
+        and accepted_email_domains.tld is not null,
+        false) as is_valid_email_address
+    from customers
+		left join accepted_email_domains
+        on customers.email_top_level_domain = lower(accepted_email_domains.tld)
+
+)
+
+select * from check_valid_emails
+```
+
+This type of logic can be challenging to validate. Let’s add a unit test to this model to ensure our `is_valid_email_address` logic captures all known edge cases: emails without `.`, emails without `@`, and emails from invalid domains.
+
+```yaml
```

Comment:
Hey @matthewshaver We should include a filename here so it's clear where to put this. The reference example uses `<file name='dbt_project.yml'>` if that helps

Reply:
Good catch! Thank you